code/game/machinery/vending/vendor_types/crew/sea.dm

```diff
@@ -4,14 +4,20 @@ GLOBAL_LIST_INIT(cm_vending_gear_sea, list(
 		list("PERSONAL PRIMARY (CHOOSE 1)", 0, null, null, null),
 		list("M41A MK1 Pulse Rifle", 0, /obj/item/weapon/gun/rifle/m41aMK1, MARINE_CAN_BUY_ESSENTIALS, VENDOR_ITEM_REGULAR),
 
-		list("SPARE AMMUNITION", 0, null, null, null),
-		list("M41A MK1 Magazine", 9, /obj/item/ammo_magazine/rifle/m41aMK1, null, VENDOR_ITEM_REGULAR),
-
-		list("GUN ATTACHMENTS (CHOOSE 1)", 0, null, null, null),
```

Comment:
Whats with the removal of the attachment? This was put there mainly so the SEA doesn't poach attachments from req or squad prep.

Reply:
As I said in the mentor channel, the removal was because it used the attachment tag which is now used by the sidearms so that the SEA only purchases one. SEA doesn't really need an attachment on any of their guns, all of them are good without one, if you really need one in emergency you can go to a squad prep or req.